Assignment3/posnums.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
```

Comment:
you program doesn't seem to tell us whether the second number is prime sometimes (for example if you input the numbers 10 and 1) and sometimes print the second number's prime/not write status sometimes (for example, 29 and 2). To avoid this in the future, you should test your code with all combinations!